Makefile

```diff
@@ -93,7 +88,16 @@ E2E_TEST_FLAGS = "--skip-deletion=false" -timeout 15m # See README.md, default g
 # Default image-build is to not use local odh-manifests folder
 # set to "true" to use local instead
 # see target "image-build"
-IMAGE_BUILD_FLAGS = --build-arg USE_LOCAL=false
+IMAGE_BUILD_FLAGS ?= --build-arg USE_LOCAL=false
+
+# Read any custom variables overrides from a .env file.  This will only be read if it exists in the 
+# same directory as this Makefile.  Variables can be specified in the standard format supported by 
+# GNU Make since include process Makefiles
+# Standard variables override would include anything you would pass at runtime that is different
+# from the defaults specified in this file
+OPERATOR_MAKE_ENV_FILE = .env
+-include $(OPERATOR_MAKE_ENV_FILE)
```

Comment:
should we move this block in the top of Makefile?

Reply:
I would say no. I would vote for the idea to have variables defaults set above it and then override them in the include. Then get rid of all the '?=' to ensure no accidental initialization from process environment.

Reply:
I just do not see any sense of introducing a variable here. Is it used somewhere else? If you want emphasize that it is Makefile, may be it is enough to add .mk suffix.

Reply:
That is a good question. Originally, I had it at the top but that would restrict overrides to any variable below that utilizes `VAR ?= VALUE`.  If there is no real value in putting it at the bottom then I have no issues moving it back to the top for visibility